### PR TITLE
Feature/angular boilerplate

### PR DESCRIPTION
**Pull request by CloudNiner**

### CODE REVIEW

**Comment by CloudNiner** [comment]
This is still very boilerplate, but it should be enough to unblock @flibbertigibbet. I added a blank nih.profiles module.

Things I'm still going to do as part of this card:
- Set up the views defined in the wireframes using ui-router
- ngARIA
- Tweak jshint settings
- Install compass, bower, grunt-cli as part of the ansible provisioning process

File: src/nih_wayfinding/Gruntfile.js [hunk 1]
```diff
@@ -0,0 +1,444 @@
...
+          '<%= yeoman.app %>/images/{,*/}*.{png,jpg,jpeg,gif,webp,svg}'
+        ]
+      }
+    },
+
+    // The actual grunt server settings
+    connect: {
+      options: {
+        port: 9000,
+        // Change this to '0.0.0.0' to access the server from outside.
```
Comment (kshepard) [inline src/nih_wayfinding/Gruntfile.js]: Update this comment

File: src/nih_wayfinding/Gruntfile.js [hunk 2]
```diff
@@ -0,0 +1,444 @@
...
+    // Performs rewrites based on filerev and the useminPrepare configuration
+    usemin: {
+      html: ['<%= yeoman.dist %>/{,*/}*.html'],
+      css: ['<%= yeoman.dist %>/styles/{,*/}*.css'],
+      options: {
+        assetsDirs: ['<%= yeoman.dist %>','<%= yeoman.dist %>/images']
+      }
+    },
+
+    // The following *-min tasks will produce minified files in the dist folder
```
Comment (kshepard) [inline src/nih_wayfinding/Gruntfile.js]: Any reason these are commented out?

**Comment by kshepard** [comment]
I'm assuming that's the Yeoman favicon. Let's ditch it before it gets cached in our browsers.

File: src/nih_wayfinding/test/.jshintrc [hunk 3]
```diff
@@ -0,0 +1,36 @@
+{
+  "node": true,
+  "browser": true,
+  "esnext": true,
+  "bitwise": true,
+  "camelcase": true,
+  "curly": true,
+  "eqeqeq": true,
+  "immed": true,
+  "indent": 2,
```
Comment (kshepard) [inline src/nih_wayfinding/test/.jshintrc]: Should be a 4

**Comment by kshepard** [comment]
+1

File: src/nih_wayfinding/Gruntfile.js (hunk 2 above)
Comment (CloudNiner) [inline src/nih_wayfinding/Gruntfile.js]: That's part of the generated boilerplate, it shouldn't apply to us. Going to remove it.

File: src/nih_wayfinding/test/.jshintrc (hunk 3 above)
Comment (CloudNiner) [inline src/nih_wayfinding/test/.jshintrc]: crap. missed one.

**Pull request closed without merging**